journal/modernized/03-chapter-2.adoc

```diff
@@ -155,6 +155,8 @@ Sell all thou hast--Oft has this gracious word
 Within my bosom secretly been heard,
 To part with all.
 ____
+[.embedded-content-document.letter]
```

Comment:
Hey Maija, this is good... but it needs one blank space between lines 157 and 158. The `[.embedded-content-document.letter]` tags have to have a blank space both before them, and then also after the -- right below them. You have the right space below the --, it just needs the space added above.👍🏻